lib/pdf_fill/forms/form_helper.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'date'
+
+module PdfFill
+  module Forms
+    module FormHelper
+      def split_ssn(veteran_social_security_number)
+        return if veteran_social_security_number.blank?
+
+        veteran_social_security_number = veteran_social_security_number.tr('^0-9', '')
+
+        split_ssn = {
+          'first' => veteran_social_security_number[0..2],
+          'second' => veteran_social_security_number[3..4],
+          'third' => veteran_social_security_number[5..8]
+        }
+
+        split_ssn
+      end
+
+      # VA file number can be up to 10 digits long; An optional leading 'c' or 'C' followed by
+      # 7-9 digits. The file number field on the 4142 form has space for 9 characters so trim the
+      # potential leading 'c' to ensure the file number will fit into the form without overflow.
+      def extract_va_file_number(va_file_number)
+        return va_file_number if va_file_number.blank? || va_file_number.length < 10
+        va_file_number.sub(/^[Cc]/, '')
+      end
```

Comment:
the `centralMailVaFile` definition in the json schema does not accepting a leading `c`, only a 8 or 9 digit number